.github/workflows/integration_test.yaml

```diff
@@ -6,6 +6,7 @@ on:
 
 jobs:
   extra-args:
+    timeout-minutes: 120
     runs-on: ubuntu-latest
```

Comment:
this times the `extra-args` job.  You wanna time the `integration-tests` job on line 36

Reply:
True. Maybe my Vscode is lying let's see what CI says...